ui/app/mainui/Handlers/SendModalHandler.qml

```diff
@@ -1072,10 +1099,16 @@ QtObject {
                     }
 
                     fiatFees: {
-                        const feeSymbol = Utils.getNativeTokenSymbol(simpleSendModal.selectedChainId)
-                        const feeToken = SQUtils.ModelUtils.getByKey(root.plainTokensBySymbolModel, "key", feeSymbol)
-                        const feeTokenPrice = !!feeToken ? feeToken.marketDetails.currencyPrice.amount: 1
-                        return root.fnFormatCurrencyAmount(feeTokenPrice*decimalTotalFees, root.currentCurrency).toString()
+			// Use GWEI fees when market data is not available
```

Comment:
not sure if it's github playing tricks or we've got some weird indent here as well

Reply:
nah, I believe my editor doesn't like qml indentation :) will find the issue